### ISSUE TITLE
OS101 Modue 2

### ISSUE THREAD

**Issue by kyuenjpl** [nasa/Transform-to-Open-Science#604]
When starting the module at lesson 1, the learning objectives do not appear. I have tried going in and out of full screen, refreshing the page and exiting/reentering module. I am using Firefox as a browser.
<img width="2560" alt="Screenshot 2024-01-04 at 1 52 51 PM" src="https://github.com/nasa/Transform-to-Open-Science/assets/53318285/d4d74447-f119-4313-b15c-8e3d9a38a4a2">

**Comment by bressler95tops** [comment]
@kyuenjpl thank you, this does appear to be a curriculum specific issue, I will communicate this to them for you.

**Comment by katblanchette** [comment]
@bressler95tops - I believe this is tech issues with the MOOC. I will share with RG.

**Comment by bressler95tops** [comment]
@katblanchette yes this is specific to the MOOC, I have only added the [OS101 Status: Curriculum](url) label to indicate that where it needs to be addressed
